source/PlayState.hx

```diff
@@ -1,12 +1,18 @@
 package;
 
+import flixel.FlxSprite;
 import flixel.FlxState;
 
 class PlayState extends FlxState
 {
+	var hero:FlxSprite;
+
 	override public function create()
 	{
 		super.create();
+
+		hero = new FlxSprite(300, 300);
```

Comment:
So, instead of creating an FlxSprite here with
```
hero = new FlxSprite(300, 300);
```
we want to create a Hero object with
```
hero = new Hero(300, 300);
```